Packs/Threat_Crowd/Integrations/ThreatCrowdV2/ThreatCrowd_v2.py

```diff
@@ -0,0 +1,270 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any, List
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+''' CONSTANTS '''
+VENDOR = 'Threat Crowd'
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API.
+    """
+
+    def __init__(self, base_url, verify, proxy, reliability):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+        self.reliability = reliability
+
+    def http_request(self, url_postfix: str, **kwargs) -> dict:
+        """
+        Overrides Base client request function, retrieves and adds to headers access token before sending the request.
+
+        Returns:
+            requests.Response: The http response
+        """
+
+        return super()._http_request('GET', url_postfix, params=kwargs)
+
+
+''' HELPER FUNCTIONS '''
+
+
+def _get_dbot_score(json_res: dict):
+    """Gets a json response and calculate the dbot score by the response"""
+
+    if json_res.get('response_code') == '1':
+        votes = json_res.get('votes')
+        if votes == -1:
+            return Common.DBotScore.BAD
+        elif votes == 0:
+            return Common.DBotScore.SUSPICIOUS
+        elif votes == 1:
+            return Common.DBotScore.GOOD
+
+    return Common.DBotScore.NONE
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def ip_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    api_url = 'ip/report/'
```

Comment:
as it is used once, i think it is more readable to be passed directly in `client._http_request`
same for all occurrences